static/js/main.js

```diff
@@ -5,11 +5,40 @@ const body = document.querySelector('body');
 let maxPagesReached = false;
 let loadingData = false;
 let selectedCountry;
-const darkModeIsOn = localStorage.getItem('dark-mode') === 'enabled';
 
-localStorage.getItem('dark-mode') === null ?
-    localStorage.setItem('dark-mode', 'disabled') :
-    body.classList.toggle('dark-mode', localStorage.getItem('dark-mode') !== 'disabled');
+const setCookie = (cookieName, cookieValue, expirationDays) => {
```

Comment:
Waar we het eerder over hebben gehad, dit lijkt me goed voorbeeld, cookies kan je beter in een aparte file zetten. Hiermee kan je deze includen waar nodig, omdat het generiek is, kan je deze meerder plekken gebruiken.